packages/form-js-editor/src/FormEditor.js

```diff
@@ -151,14 +160,23 @@ export default class FormEditor {
   }
 
   detach() {
+    this._detach();
```

Comment:
How about if we don't use any flags and emit the event where we want?

```suggestion
    this._emit('detach');
    this._detach();
```

Reply:
The behavior would be different. The event would be fired even if there wasn't a detach.